### ISSUE TITLE
The Test Results

### ISSUE THREAD

**Issue by Jason-ic** [woo525/ParamISP#1]
Hello, thank you very much for your excellent work. I am very interested in your work as well. However, during my reproduction process, I directly used the fine-tuned model you provided to test on S7, but I did not achieve the results mentioned in the paper. I believe the issue lies in the dataset split. Additionally, it seems that the two dataset split txt files for S7 in the project only contain partial samples. Could you please provide the complete training, validation, and test set txt files? Below are the results I obtained using the first 150 and the last 150 images from S7, respectively:

           L1               0.01285370159894228
         ms-ssim            0.9370995163917542
          psnr              35.476016998291016
          ssim              0.9080767631530762

           L1              0.012729881331324577
         ms-ssim             0.938057541847229
          psnr              35.623348236083984
          ssim              0.9203691482543945
          
but in your paper is 35.20. At the same time, in your test code, there are several testing methods: full, center, 5, and 13. Could you please let me know which method you used for testing? I am looking forward to your response.
